azure/packages/external-controller/package.json

```diff
@@ -75,10 +75,10 @@
 		"jest": "^26.6.3",
 		"jest-environment-puppeteer": "^4.3.0",
 		"jest-junit": "^10.0.0",
-		"jest-puppeteer": "^4.3.0",
+		"jest-puppeteer": "^6.2.0",
 		"prettier": "~2.6.2",
 		"process": "^0.11.10",
-		"puppeteer": "^1.20.0",
+		"puppeteer": "^17.1.3",
```

Comment:
I see these were updated in several places... are they necessary as part of bumping build-common? If so, can we update the PR description to clarify why?

Reply:
Added description.